pkg/server/container_create_unix.go

```diff
@@ -69,7 +70,9 @@ const (
 // CreateContainer creates a new container in the given PodSandbox.
 func (c *criService) CreateContainer(ctx context.Context, r *runtime.CreateContainerRequest) (_ *runtime.CreateContainerResponse, retErr error) {
 	config := r.GetConfig()
-	log.G(ctx).Debugf("Container config %+v", config)
+	log.G(ctx).
+		WithField("config", fmt.Sprintf("%+v", c.Scrub(*config))).
```

Comment:
The comment on the Scrub function indicates that we *shouldn't* be passing in a pointer but we are here.

Reply:
Here `config` is a ` *ContainerConfig`, so it should be dereferenced and passed as a value to `Scrub`, right?

Reply:
Oh oh right, my bad